### PR TITLE
Review of Data Hub Share Data How To

### PR DESCRIPTION
**Pull request by MarkvanMents**

### CODE REVIEW

File: content/data-hub/general/share-data.md [hunk 1]
```diff
@@ -31,25 +30,9 @@ Before starting this how-to, make sure you have completed the following prerequi
...
-
-In Studio Pro, when you have an open project, the **Data Hub** pane will be displayed on the right. 
-
-![data hub pane](attachments/share-data/data-hub-pane-empty.png)
-
-If you do not see the above, click  **View** > **Data Hub** to display the **Data Hub** pane. 
-
-![](attachments/share-data/view-data-hub.png)
-
-{{% todo %}}[**AD: this section now only states info about opening the Data Hub pane, thus making the section superfluous. Is section 7 the best place to move this info so as to delete this section?**]{{% /todo %}}
```
Comment (MarkvanMents) [inline content/data-hub/general/share-data.md:48]: I've moved the relevant stuff to section 7.
A How-to should contain instructions on how to do things, and some explanation of why. this seems much more reference material.

File: content/data-hub/general/share-data.md [hunk 2]
```diff
@@ -66,110 +49,122 @@ Follow these steps to create a simple app in Studio Pro whose data you use in an
...
+5. Click **OK** to see the entity and attributes in the domain model.
+6. Right-click the entity and from the menu,  select **Generate overview pages**.
+7. In the **Generate pages** dialog box, select **Atlas_Default(Atlas_UI_Resources)** as the **Content layout** and click **OK**. Overview pages for the new entity will be added in the **OverviewPages** folder of **MyFirstModule**.
 
 	![](attachments/share-data/overview-pages-for-customer-entity.png)
 
-7.  In the **Project Explorer**, double-click **Home_Web** to open the **Home_Web** page. From the **Project Explorer** drag **Customer_Overview** into the **Auto-fill** container under the "Welcome" banner. You have now created a simple app with the entity **Customer** and a web page where you can add data and view and edit details for this entity. Go ahead and customize your Home page further by changing the banner text.
+7.  In the **Project Explorer**, double-click **Home_Web** to open the **Home_Web** page.
```
Comment (MarkvanMents) [inline content/data-hub/general/share-data.md:58]: Split multiple steps into individual steps.

File: content/data-hub/general/share-data.md [hunk 3]
```diff
@@ -180,41 +175,46 @@ You are now going to create a new app and consume the data you have added to the
...
 1. When the app has successfully been deployed, click **View** to open the app in the browser.
 2. Click **Customer Overview**.
-3.  The overview page displays the list of the customers that you entered in the **{yourname}CustomerServiceApp** app. 
 
-	{{% alert type="info" %}}Notice that, as this page displays data shared from another app, there are no buttons for adding or changing this data.{{% /alert %}}
+	The overview page displays the list of the customers that you entered in the **{yourname}CustomerServiceApp** app. 
+
+	{{% alert type="info" %}}As this page displays data shared from another app, there are no buttons for adding or changing this data.{{% /alert %}}
+
+{{% alert type="warning" %}}
```
Comment (MarkvanMents) [inline content/data-hub/general/share-data.md:207]: Added a warning here. It happened to me when I went off to a meeting and then came back to this section.

File: content/data-hub/general/share-data.md [hunk 4]
```diff
@@ -180,41 +175,46 @@ You are now going to create a new app and consume the data you have added to the
...
 
-## 9 Seeing Changes in Data in the Consuming App
+## 8 Seeing Changes in Data in the Consuming App
 
 To see an example of consumed data being updated when data is changed in the originating app, follow these steps:
 
 1. Open both apps that have been created in this how-to in separate browser windows and display them side by side. 
 2. Make some changes to the customer list in **{yourname}CustomerServiceApp** by adding a few more customers to the list and editing some existing entries.
-3. Refresh the **{yourname}CustomerActionsApp** window by clicking **View** or **Search** to see the changes in the data displayed. 
+3. Refresh the **{yourname}CustomerActionsApp** window by doing a **Search** to see the changes in the data displayed.
```
Comment (MarkvanMents) [inline content/data-hub/general/share-data.md:217]: View didn't seem to do anything here - I think search is better. And the users don't need a choice in a how to.

File: content/data-hub/general/share-data.md [hunk 5]
```diff
@@ -233,26 +233,18 @@ You will now learn how to do the following:
 
 You can view the two apps that you have created in the Data Hub Landscape and see the associations by following these steps:
 
-1. You can find the service that you have published in the Data Hub Catalog from the [Data Hub](https://hub.mendix.com/#/home) home page. Alternatively, you can also navigate directly to the service from the Studio Pro **Data Hub** pane.
-2.  With the service selected in the Data Hub, click the **Landscape** tab to see  a graphical representation of the context of the selected item in the **Search Pane**:
```
Comment (MarkvanMents) [inline content/data-hub/general/share-data.md:237]: This didn't work for me the first time. So I've changed the instructions a bit to make sure it always works.

**Comment by Adam-Dupaski** [comment]
@ilagordhan please review this as soon as possible so we can merge this into the main Data Hub branch to publish tomorrow morning.

**Pull request merged**